subsys/bluetooth/services/Kconfig.gap_default

```diff
@@ -0,0 +1,12 @@
+# GATT Generic Access Service - default implementation
+#
+# Copyright (c) 2025 Koppel Electronic
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_GAP_DEFAULT
```

Comment:
This config name is too generic. The GAP Service is only one component of GAP. I suggest something like `BT_GAP_SVC_DEFAULT_IMPL`.